packages/zsteg/PKGBUILD

```diff
@@ -0,0 +1,43 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=zsteg
+_gemname=zsteg
```

Comment:
_gemname is identical to pkgname so remove _gemname and replace it everywhere with pkgname

Reply:
Fixed